src/db/dal/user.ts

```diff
@@ -28,10 +134,42 @@ export const searchUsers = async (pageSize: number, pageIndex: number) => {
     };
 };
 
-export const getUserById = async (keycloak_id: string): Promise<IUserOuput> => {
+export const getProfileImageUploadPresignedUrl = async (keycloak_id: string) => {
+    if (!S3Client) {
+        return {
+            s3Key: undefined,
+            presignUrl: undefined,
+        };
+    }
+
+    const s3Key = `${keycloak_id}.${profileImageExtension}`;
+    const presignUrl = S3Client.getSignedUrl('putObject', {
+        Bucket: profileImageBucket,
+        Key: s3Key,
+        Expires: 60 * 5,
+        ContentType: 'image/jpeg',
```

Comment:
we should leverage `profileImageExtension` and do something like: 'image/${profileImageExtension}'
Otherwise, why bother with a variable name for jpeg at all?